src/api/cloudinary.js

```diff
@@ -0,0 +1,27 @@
+const cloudName = 'dlxrqqprn'
```

Comment:
Why do we want to do unsigned cloudinary uploads? Why can't we do signed uploads via our server?

Reply:
> There needs to be a "back to event" link at the top.

Will address this in the next PR where I actually add a link to the migration page and style things up

Reply:
I'm fine with trying to address cloudinary uploads via the survey in a separate PR and have it. It was just easier to get it working client-side only and since it was only for event images it doesn't seem like a big deal for now